_data/matrix/Bitkit.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Bitkit"
+category: "Software Wallet (L1&L2)"
+keyfeatures: "Your ultimate Bitcoin toolkit. Self-custodial BTC and LN payments. Widgets, pay-to-contacts and more"
```

Comment:
Since the Feature Matrix is only focused on a given set of features, I added the column **Key Features** / Use Cases.  The objective here was to allow for product teams to highlight things which might differentiate their product, especially if any of their key features are not in the (current) scope of the matrix.